src/main/scala/nl.knaw.dans.easy.multideposit/actions/AddBagToDeposit.scala

```diff
@@ -34,25 +37,28 @@ import scala.util.{ Failure, Try }
 
 class AddBagToDeposit extends DebugEnhancedLogging {
 
-  def addBagToDeposit(depositId: DepositId, created: DateTime)(implicit input: InputPathExplorer, stage: StagingPathExplorer): Try[Unit] = {
+  def addBagToDeposit(depositId: DepositId, created: DateTime, base:String)(implicit input: InputPathExplorer, stage: StagingPathExplorer): Try[Unit] = {
     logger.debug(s"construct the bag for $depositId with timestamp ${ created.toString(ISODateTimeFormat.dateTime()) }")
 
-    createBag(depositId, created) recoverWith {
+    createBag(depositId, created, base) recoverWith {
       case NonFatal(e) => Failure(ActionException(s"Error occured in creating the bag for $depositId", e))
     }
   }
 
-  private def createBag(depositId: DepositId, created: DateTime)(implicit input: InputPathExplorer, stage: StagingPathExplorer): Try[Unit] = Try {
+  private def createBag(depositId: DepositId, created: DateTime, base:String)(implicit input: InputPathExplorer, stage: StagingPathExplorer): Try[Unit] = Try {
     val inputDir = input.depositDir(depositId)
     val stageDir = stage.stagingBagDir(depositId)
 
     val metadata = new BagitMetadata {
       add("Created", created.toString(ISODateTimeFormat.dateTime()))
     }
 
+    metadata.add("Is-Version-Of", base )
+
     if (Files.exists(inputDir)) {
       inputDir.copyDir(stageDir)
       metadata.add("Bag-Size", formatSize(calculateSizeOfPath(inputDir)))
+
```

Comment:
unnecessary newlines here and on lines 102 and 103

Reply:
That's right. In IntelliJ you should see this empty line at the end of each file.